tests/JakubOnderka/PhpVarDumpCheck/CustomFunctionTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+use JakubOnderka\PhpVarDumpCheck;
+
+class CustomFunctionTest extends PHPUnit_Framework_TestCase
+{
+    public function __construct()
+    {
+
+    }
+
+
```

Comment:
What about add test for none function founded? (I mean when code does not includes `functionName1`) It coverage all cases